systemd/manager.go

```diff
@@ -106,6 +106,13 @@ func (m *systemdUnitManager) Load(name string, u unit.UnitFile) error {
 	if err != nil {
 		return err
 	}
+	if u.HasReverseDependencies() {
```

Comment:
Well, you tell me ;-)

I'm assuming if it's empty, systemd will enable it just fine? And if it's invalid, it will surely bomb -- which I'd say is the right thing to do...